install/0001_00_cluster-version-operator_03_service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Is the `0001_` prefix intentional?  I guess that means "first after all of the ordered objects with [special runlevels][1]"?

[1]: https://github.com/openshift/cluster-version-operator/blob/d2914283326951a6a6035a42b4b9194aa9105bbf/docs/dev/operators.md#how-do-i-get-added-as-a-special-run-level